fireant/database/mysql.py

```diff
@@ -49,13 +47,39 @@ def __init__(self, host='localhost', port=3306, database=None,
         self.password = password
         self.charset = charset
 
-    def connect(self):
+    def _get_connection_class(self):
         import pymysql
 
-        return pymysql.connect(host=self.host, port=self.port, db=self.database,
-                               user=self.user, password=self.password,
-                               charset=self.charset,
-                               cursorclass=pymysql.cursors.Cursor)
+        class MySQLConnection(pymysql.connections.Connection):
```

Comment:
does this need to be defined within the function?

Reply:
No i'll remove it. It stayed nested because of a refactor I did, and I forgot to take it out :) will fix.

Reply:
Actually, ignore my last comment. It does - the reason is users can install fireant without the mysql extra. If they do this, pymysql is not imported, so we need to nest it inside a function to prevent the app from trying to import the module. I'll add a comment!